.travis.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     - stage: test
       script: ./gradlew check -i
     - stage: Push docker images
```

Comment:
Instead of making this a stage, I think you can also put it under the `deploy` key, like here: https://github.com/xenit-eu/alfresco-gradle-sdk/blob/master/.travis.yml#L13-L18

Reply:
I tried, but I had troubles. Why is it better?

Reply:
Deploys will never run for pullrequests